tic_tac_toe/game.py

```diff
@@ -33,7 +45,9 @@ def _board_is_full(board):
 
     :param board: Game board.
     """
-    pass
+    flat_board = [marker for row in board for marker in row]
+    status = [True if place != '-' else False for place in flat_board]
```

Comment:
Instead of using an extra list comprehension and all you could just `return '-' in flat_board` to simplify things a bit.